libraries/kotlinandroidlibrary/build.gradle

```diff
@@ -3,19 +3,31 @@ apply plugin: "kotlin-android"
 apply plugin: "kotlin-android-extensions"
 
 androidExtensions {
-  experimental = true
+    experimental = true
 }
 
 dependencies {
-  api deps.external.kotlinStdlib
-  testImplementation deps.test.junit
-  testImplementation deps.test.kotlinTest
+    api deps.external.kotlinStdlib
+    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+    implementation 'com.android.support:design:27.1.1'
+    testImplementation deps.test.junit
+    testImplementation deps.test.kotlinTest
+
+    androidTestImplementation deps.test.junit
```

Comment:
This is a breaking change/behavior for okbuck consumers

We should confirm that `androidTest` configurations do not extend from `test` configurations in AGP

Reply:
They shouldn't. There are example of libraries you can use on jvm tests that you can't on android runtime, as ART doesn't have some java packages (such as java.nio.file) But I'll confirm

Reply:
But why is this a breaking chage/behavior? Do we assume they do?

Reply:
Before some dependencies were picked up from `test` configs, but now they wont be. This might require some consumers to add such dependencies in the `androidTest` configurations explicitly. So its a potential breaking change

Reply:
Hum, I've confirmed and test and androidTest configs do not extend from each other. So the right thing to do (and consumers should be doing that already, if they wanna make their project "gradle supported" too), is to explicitly put those configs. WDYT?

Reply:
Ya I agree, its the right thing to do. We just need to call out in the release notes that the behavior changed and its still breaking for someone that relied on the wrong behavior before :)

Just making a note to add it to the release notes for myself. The change itself is fine